README.md

```diff
@@ -0,0 +1,60 @@
+# Todo-List
+
+
+> This is a simple Todo List
+
+![screenshot](/img/todo.png ) 
+
+## Built With
+
+- HTML5
+
+- CSS
+
+- Boostrap
+
+- Javascript
+
+- Visual Studio Code
+
+- Git & Github
+
+
+## Live Server
+[Todo- List](https://hopeful-brown-dda6d4.netlify.app/)
```

Comment:
your live demo doesn't work please try and fix that.

Reply:
Thanks

Reply:
Welcome, Peter.

Reply:
Can you check the live view here please https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/
The edit feature on the li item and not the icon

Reply:
Oh. Yes. You are totally right. please forgive the mistakeke